src/services/timer.js

```diff
@@ -1,7 +1,7 @@
 export const PERIODS = {
-  work: { id: 'work', mins: 25, secs: 0 },
-  shortBrk: { id: 'short-brk', mins: 5, secs: 0 },
-  longBrk: { id: 'long-brk', mins: 15, secs: 0 },
+  work: { id: 'work', mins: 0, secs: 5 },
+  shortBrk: { id: 'short-brk', mins: 0, secs: 2 },
+  longBrk: { id: 'long-brk', mins: 0, secs: 3 },
```

Comment:
Why are you changing these values?

Reply:
I was running some tests on the code so I had to reduce the time of the counter to speed up the checking process, and when I finished I **forgot** to reset the counter to its default.